src/test/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposerTest.java

```diff
@@ -515,4 +625,111 @@ public void composeUnaryRpcMethodSampleCode_methodReturnVoid() {
             "}");
     Assert.assertEquals(results, expected);
   }
+
+  // ===================================Unary Paged RPC Method Sample Code ======================//
+  @Test
+  public void composeUnaryPagedRpcMethodSampleCode_multipleMethodArguments() {
+    TypeNode inputType =
+        TypeNode.withReference(
+            VaporReference.builder()
+                .setName("ListContentRequest")
+                .setPakkage(PACKAGE_NAME)
+                .build());
+    TypeNode outputType =
+        TypeNode.withReference(
+            VaporReference.builder()
+                .setName("ListContentResponse")
+                .setPakkage(PACKAGE_NAME)
+                .build());
+    TypeNode resourceNameType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(List.class)
+                .setGenerics(ConcreteReference.withClazz(String.class))
+                .build());
+    List<MethodArgument> arguments =
+        Arrays.asList(
+            MethodArgument.builder()
+                .setName("resourceName")
+                .setType(resourceNameType)
+                .setField(
+                    Field.builder()
+                        .setName("resourceName")
+                        .setType(resourceNameType)
+                        .setIsRepeated(true)
+                        .build())
+                .build(),
+            MethodArgument.builder()
+                .setName("filter")
+                .setType(TypeNode.STRING)
+                .setField(Field.builder().setName("filter").setType(TypeNode.STRING).build())
+                .build());
+    Method method =
+        Method.builder()
+            .setName("ListContent")
+            .setMethodSignatures(Arrays.asList(arguments))
+            .setInputType(inputType)
+            .setOutputType(outputType)
+            .setIsPaged(true)
+            .build();
+    TypeNode repeatedResponseType =
+        TypeNode.withReference(
+            VaporReference.builder().setName("Content").setPakkage(PACKAGE_NAME).build());
+
+    String results =
+        SampleCodeWriter.write(
+            ServiceClientSampleCodeComposer.composeUnaryPagedRpcMethodSampleCode(
+                method, arguments, clientType, resourceNames, repeatedResponseType));
+    String expected =
+        LineFormatter.lines(
+            "try (EchoClient echoClient = EchoClient.create()) {\n",
+            "  List<String> resourceName = new ArrayList<>();\n",
+            "  String filter = \"filter-1274492040\";\n",
+            "  for (Content element : echoClient.listContent(resourceName, filter).iterateAll()) {\n",
+            "    // doThingsWith(element);\n",
+            "  }\n",
+            "}");
+    Assert.assertEquals(results, expected);
```

Comment:
Why `Assert.assertEquals` instead of just `assertEquals`?

Reply:
I guess it added by accident. Removed.